bf2fc6cc711aee1a0c2a/kas-fleet-manager
# feat: Deprecate instanceType for cloud type region endpoint

## Description
We now have instanceType argument in the cloud provider region but the API also returns RegionCapacityListItem which has instance_types. Currently capacity field is returning all instance types despite the argument being passed So our logic can check if the region is enabled, Check if instance type is supported and then check if we have capacity in that region Since the capacity field already returns the instance types, we no longer need to return the `supported_instance_types` as well. The `instance_type` filter can be removed since it's not being used.

Jira: https://issues.redhat.com/browse/MGDSTRM-7493

## Verification Steps
1. Run make run/docs
2. Go to /regions endpoint
3. Can see that instance_type filter is deprecated 
4. Scroll to bottom can see that in the CloudRegion schema that supported_instance_types is deprecated

## Checklist (Definition of Done)
